src/Executor/Setup.cs

```diff
@@ -0,0 +1,290 @@
+        public Dictionary<string, Command> Commands { get; set; }
+        public Dictionary<string, Tool> Tools { get; set; }
+        public Dictionary<string, string> SettingParameters { get; set; }
+        public string Os { get; set; }
+        public string ConfigurationFilePath { get; set; }
+
+
+        private string FindSettingValue(string valueToFind)
+        {
+            if (Settings.ContainsKey(valueToFind))
+            {
+                return Settings[valueToFind].DefaultValue;
+            }
+            return null;
+        }
+
+        private string FindSettingType(string valueToFind)
+        {
+            if (Settings.ContainsKey(valueToFind))
+            {
+                return Settings[valueToFind].ValueType;
+            }
+            return null;
+        }
+        
+        public void prepareValues(string os, Dictionary<string, string> parameters, string configFile)
+        {
+            SettingParameters = new Dictionary<string, string>(parameters);
+            Os = os;
+            ConfigurationFilePath = configFile;
+        }
+
+        public int ExecuteCommand(string commandSelectedByUser)
+        {
+            string[] commandToRun = BuildCommand(commandSelectedByUser);
+            if(commandToRun != null)
+            {
+                Console.ForegroundColor = ConsoleColor.DarkYellow;
+                Console.WriteLine("Running: {0} {1}", commandToRun[0], commandToRun[1]);
+                Console.ResetColor();
+
+                Run runCommand = new Run();
+                int result = runCommand.ExecuteProcess(commandToRun[0], commandToRun[1]);
+                if (result == 0)
+                {
+                    Console.ForegroundColor = ConsoleColor.Green;
+                    Console.WriteLine("Build Succeeded.");
+                }
+                else
+                {
+                    Console.ForegroundColor = ConsoleColor.Red;
+                    Console.WriteLine("Build Failed.");
+                }
+                Console.ResetColor();
+
+                return result;
+            }
+            return 1;
+        }
+
+        public string[] BuildCommand(string commandSelectedByUser, Dictionary<string, string> parameters = null)
+        {
+            if (!Commands.ContainsKey(commandSelectedByUser))
+            {
+                Console.WriteLine("Error: The command {0} is not specified in the Json file.", commandSelectedByUser);
+                return null;
+            }
+
+            Command commandToExecute = Commands[commandSelectedByUser];
+            string toolName = GetTool(commandToExecute, Os, ConfigurationFilePath);
+            if (string.IsNullOrEmpty(toolName))
+            {
+                Console.WriteLine("Error: The process {0} is not specified in the Json file.", commandToExecute.ToolName);
+                return null;
+            }
+
+            string commandParameters = string.Empty;
```

Comment:
This value isn't actually used in this code block, so it doesn't need to be initialized, or even declared at this scope.